### ISSUE TITLE
Always default base-address

### ISSUE THREAD

**Issue by gvtienen** [WSNhub/MySensorsGateway#7]
Since last upgrade (10 commits) I am not able to set the private base-address for the radio
(whatever setting I do, I am always getting the default address)

**Comment by alainmaes** [comment]
Tried it, and it is updated with the base-address command. It also shows the correct address in the info command and on the status page.

base-address default
RF base address    : a8a8e1fc00

base-address private
RF base address    : 00dc0f2800

**Comment by gvtienen** [comment]
I have tried it again, this is the log taken from the console :
MySensorsGateway > Received full Command line, size = 20,cmd = base-address private
CommandExecutor : executing command base-address
Returning Delegate for base-address 

state: 5 -> 0 (0)
rm 0
TCP connection error: -8
del if0
bcn 0
del if1
usl
MySensorsGateway > Dbg  230.  6 : Wifi client got disconnected (8)
mode : sta(5c:cf:7f:81:d2:e8)
add if0
mode : sta(5c:cf:7f:81:d2:e8) + softAP(5e:cf:7f:81:d2:e8)
add if1
dhcp server start:(ip:192.168.4.1,mask:255.255.255.0,gw:192.168.4.1)
Fatal exception 9(LoadStoreAlignmentCause):
epc1=0x40100858, epc2=0x00000000, epc3=0x00000000, excvaddr=0x00000003, depc=0x00000000

 ets Jan  8 2013,rst cause:2, boot mode:(3,7)

load 0x40100000, len 1364, room 16 
tail 4
chksum 0x22
load 0x3ffe8000, len 660, room 4 
tail 0
chksum 0x06
csum 0x06

rBoot v1.3.0 - richardaburton@gmail.com
Flash Size:   32 Mbit
Flash Mode:   QIO
Flash Speed:  40 MHz
rBoot Option: Big flash

Booting rom 1.
don't use rtc mem data
;␔+TCP connection
timeout updating: 70 -> 65535
Server timeout updating: 40 -> 900
+TCP connection
timeout updating: 70 -> 65535
+TCP connection
timeout updating: 70 -> 65535
Server timeout updating: 40 -> 90
+TCP connection
DebugClass Instantiating
fs.start: size:256 Kb, offset:0x300000

mount res: 0

Commandhandlercommand help registered
Welcome to the Stream Command executor
Dbg    1.300 : Debug started
Commandhandlercommand info registered
Commandhandlercommand debug registered
Commandhandlercommand restart registered
Commandhandlercommand cpu registered
Commandhandlercommand apMode registered
Commandhandlercommand js registered
Commandhandlercommand rules registered
Commandhandlercommand showConfig registered
Commandhandlercommand base-address registered
Commandhandlercommand pong registered
Set AP configuration in background
Station configuration was updated to: snoa
Commandhandlercommand upgrade registered
mode : softAP(5e:cf:7f:81:d2:e8)
add if1
dhcp server start:(ip:192.168.4.1,mask:255.255.255.0,gw:192.168.4.1)
bcn 100
'/' registered
'/ipconfig' registered
'/status' registered
'/maintenance' registered
'/ajax/getSensors' registered
'/ajax/removeSensor' registered
'getSensors' registered
'setActuator' registered
'removeSensor' registered
'/mqttconfig' registered
Dbg    1.428 : No I/O expanders found
Dbg    1.431 : No RTC found
addUser: me 123

gateway started, id=0, parent=0, distance=0

AP configuration was updated on system ready event
Dbg    1.477 : Connecting...
mode : sta(5c:cf:7f:81:d2:e8) + softAP(5e:cf:7f:81:d2:e8)
add if0
f r0, scandone
state: 0 -> 2 (b0)
state: 2 -> 3 (0)
state: 3 -> 5 (10)
add 0
aid 6
cnt 

connected with snoa, channel 6
dhcp client start...
Dbg    3.644 : Wifi client got connected
ip:192.168.1.33,mask:255.255.255.0,gw:192.168.1.1
Dbg    4.607 : Got IP
Dbg    4.613 : --> I'm CONNECTED
Dbg    4.616 : Not disabling AP due to config setting
UDP LocalPort: 4097
UDP connect to 185.77.199.1:123
time updated? 1
Dbg    4.689 : Time after NTP sync: 29.03.2016 16:51:00
+TCP connection
MQTT start connection
connect to: 192.168.1.51
TcpConnection::connect result:, 0
timeout updating: 70 -> 65535
subscription '#' registered
realloc 28 -> 165
OnConnected
TCP connected
onReadyToSendData: 0
TcpClient request completed
TCP sent: 78
onReadyToSendData: 2

> MQTT status: MQTT_MSG_CONNACK (len: 4)
> TCP received: 4 bytes
> onReadyToSendData: 1
> MQTT status: MQTT_MSG_SUBACK (len: 5)
> MQTT status: MQTT_MSG_PUBLISH (len: 42)
> 21: 17

Dbg    5.593 : RX: /ESP_0081D2E8/version = MySensors gateway
TCP received: 47 bytes
onReadyToSendData: 1
TcpClient request completed
TCP sent: 2
onReadyToSendData: 2

> MQTT status: MQTT_MSG_PINGRESP (len: 2)
> TCP received: 2 bytes
> onReadyToSendData: 1
> Received full Command line, size = 4,cmd = info
> CommandExecutor : executing command info
> Returning Delegate for info 

System information : ESP8266 based MySensors gateway
Build time         : Mon Mar 28 12:57:10 CEST 2016
Version            : 3893153
Sming Version      : 2.1.0
ESP SDK version    : 1.5.2(80914727)
MySensors version  : 1.5.4

Station SSID       : snoa
Station DHCP       : TRUE
Station IP         : 192.168.1.33

Access Point Mode  : always

System Time        : 29.03.2016 16:51:26
Free Heap          : 14832
CPU Frequency      : 160 MHz
System Chip ID     : 81d2e8
SPI Flash ID       : 1640e0
SPI Flash Size     : 4194304

RF base address    : a8a8e1fc00

MySensorsGateway >
